connman/vpn/vpn-provider.c

```diff
@@ -86,7 +87,7 @@ struct vpn_provider {
 	char *type;
 	char *host;
 	char *domain;
-	int family;
+	bool family[2];
```

Comment:
As offsets to family array are defines (AF_INET6_POS etc) -> might be nice to have constant for array size too instead of hard coded 2?

Also: is initial state enforced somehow? Like by using calloc() / similar?

Reply:
The structs are initialized with the zeroing calls, so yes. And the reset is done when the provider disconnects, on both.

But yeah, I'll replace the hardcoding, good point.

Reply:
0bc55bf077789ce8551da1daddfd78ae90b693a5 remove hardcode.